app/lib/nomis_client/moves.rb

```diff
@@ -3,6 +3,8 @@
 module NomisClient
   class Moves
     class << self
+      MAX_POSSIBLE_MOVES_IN_ONE_DAY = 500
```

Comment:
@starswan what is the purpose of this constant and is 500 a reasonable value? I thought there are about ~600k moves a year which is roughly ~2k moves per day?

Reply:
It's per-institution. I introduced it as a named constant - it used to be a hard-coded value